syntax = "proto3";
package movie_reviews;

import "google/protobuf/timestamp.proto";

// Next ID: 4
message Movie {
  string title = 1;
  google.protobuf.Timestamp release_date = 2;
  string director = 3;
}
